Library/Formula/ldc.rb

```diff
@@ -12,19 +12,39 @@ class Ldc < Formula
     sha256 "6d9d60e0a1711a12e03729f273459363873286ce77977801cfbbfc517fc7af1a" => :mavericks
   end
 
+  devel do
+    url "https://github.com/ldc-developers/ldc/releases/download/v1.0.0-alpha1/ldc-1.0.0-alpha1-src.tar.gz"
+    sha256 "b656437d0d7568c5ac4ef4366376184c06013e79f3dd5a512b18ca9f20df4b63"
+    version "1.0.0-alpha1"
+  end
+
   needs :cxx11
 
   depends_on "cmake" => :build
   depends_on "llvm" => :build
+  depends_on "dmd" => :build
```

Comment:
If this is only needed for the `devel` build it should be tucked into the `devel` block above, one empty line underneath the `version` in this case.